src/mirco_contactstatus_kokkos.h

```diff
@@ -0,0 +1,39 @@
+#ifndef SRC_CONTACTSTATUS_KOKKOS_H_
+#define SRC_CONTACTSTATUS_KOKKOS_H_
+
+#include "mirco_kokkostypes_kokkos.h"
+
+namespace MIRCO
+{
+  /**
+   * @brief The aim of this function is to compute to nodes in contact in the current iteration
+   *
+   * @param[out] xvf x-coordinates of the points in contact in the previous iteration.
+   * @param[out] yvf y-coordinates of the points in contact in the previous iteration.
+   * @param[out] pf Contact force at (xvf,yvf) predicted in the previous iteration.
+   * @param[out] nf Number of nodes in contact in the previous iteration
+   * @param[in] y Solution containing force
+   * @param[in] xv0 x-coordinates of the points in contact in the previous iteration.
+   * @param[in] yv0 y-coordinates of the points in contact in the previous iteration.
+   */
+  void ComputeContactNodes(ViewVector_d& xvf, ViewVector_d& yvf, ViewVector_d& pf_d,
+      const int activeSetSize, const ViewVector_d p_d, const ViewVector_d xv0,
+      const ViewVector_d yv0);
+
+  /**
+   * @brief The aim of this function is to calulate the contact force and contact area for the
+   * current iteration. Note that the force vector
```

Comment:
```suggestion
   * @brief Calulate the contact force and contact area for the
   * current iteration.
```

The sentence `Note that the force vector` is incomplete. What is missing here? Does it need to be part of the `@brief` description or should it go into the main body of the documentation?